angular/src/lib/stepper/stepper.component.scss

```diff
@@ -0,0 +1,129 @@
+.md-stepper-container {
```

Comment:
This scss should be moved to core directory scss file.

Reply:
Done